source/ed/build_helper.cpp

```diff
@@ -730,11 +733,18 @@ void builder::finish() {
                  continue;
              }
 
-             const auto start = freq_vj->stop_time_list.front().arrival_time;
+             const auto first_st_it = freq_vj->earliest_stop_time();
```

Comment:
That's a question I forgot to ask!

I don't know if shifting start_time is necessary or not. The thinking behind was if you have a frequency vj starting at 08:00:00, you're going to set a start_time at 08:00:00 in the NTFS, even if the boarding on some st is XX minutes, and the boarding_time is prior to 08:00:00.
But for the routing we need the start_time at the smallest possible boarding_time / arrival_time. So start_time should be at 07:XX:00.
There is two ways : 
* The start_time in the NTFS should take that into account, and I remove this code,
* We need to take care about it.

And that's why I shift end_time the same way. Currently end_time represents the last possible time at the first stop_time. Now end_time represents the last possible boarding_time.